ebu_tt_live/node/distributing.py

```diff
@@ -21,16 +21,19 @@ def __init__(self, node_id, producer_carriage=None, consumer_carriage=None, **kw
         )
 
     def process_document(self, document, raw_xml=None, **kwargs):
-        if self.check_document(document) is True:
-            if raw_xml is not None:
-                data = raw_xml
+        if self.is_document(document):
```

Comment:
This looks like another change that needs to be moved into release/3.0 separately, ideally.